src/containers/SearchPage/components/SearchResultItem.tsx

```diff
@@ -72,21 +130,33 @@ const StyledText = styled(Text, {
 
 const SearchResultItem = ({ item, type }: Props) => {
   const { t } = useTranslation();
-  const contentType = type === "topic-article" ? "topic" : type;
   const mainContext = item.contexts?.[0];
 
   return (
     <StyledListElement>
-      <StyledCardRoot>
+      <StyledCardRoot variant="subtle">
         <CardImage
           alt=""
           height={200}
           src={item.img?.url ?? item.metaImg ?? ""}
           sizes={"320px"}
-          fallbackElement={<ContentTypeFallbackIcon contentType={contentType} />}
+          fallbackElement={<ContentTypeFallbackIcon contentType={type} />}
         />
+        <Metadata data-block-start-border variant={contentTypeToVariantMapping[type]}>
+          <StyledText textStyle="label.small">{t(`contentTypes.${type}`)}</StyledText>
+          {item.labels && item.labels?.length >= 1 && (
+            <StyledText textStyle="label.xsmall">
+              {t(`searchPage.includes `)}
+              {item.labels?.map((label, index, arr) => (
+                <span key={index}>
+                  {label}
+                  {index < arr.length - 1 && ", "}
+                </span>
```

Comment:
Er det noe som stopper oss fra å gjøre dette til et string-array, for så å joine det?